src/Money.php

```diff
@@ -350,15 +350,20 @@ public function divide($divisor, $roundingMode = self::ROUND_HALF_UP)
     public function allocate(array $ratios)
     {
         if (count($ratios) === 0) {
-            throw new \InvalidArgumentException('Cannot allocate to none');
+            throw new \InvalidArgumentException('Cannot allocate to none, ratios cannot be an empty array');
```

Comment:
@frederikbosch Why not using a [component exception](https://github.com/moneyphp/money/blob/master/src/Exception/Exception.php#L10) here? The `InvalidArgumentException` must be created too.